files/init/start.sh

```diff
@@ -4,7 +4,7 @@
 # act as an entry point to startup the rest of the hobby-hub utilities (as needed).
 
 # Start program manager
-sudo python3 /etc/hobby-hub/program_manager.py /etc/hobby-hub/test_programs/ > /etc/hobby-hub/out/running_programs.txt
+sudo python3 /etc/hobby-hub/program_manager.py /home/dev/myprograms/ > /etc/hobby-hub/out/running_programs.txt
```

Comment:
@anthony-bartman @Ivendrakeo This just allows the users to write their programs in the home directory, rather than running the test programs from the test directory

Reply:
so the way I had things setup with the cmd line script was to use /etc/hobby-hub/test_programs as the program directory. then they would develop in /home/dev/myprograms and transfer then to /etc/hobby-hub/tes_programs using the utility and that would guarntee they would be run on startup. 

Think about it. your using /etc/home/dev/ as your workspace and what if your crafting a program that you don't want to run? you can still use the cmd-line utility to debug from your workspace, but when you acutally want it to run on startup, you should use the utility to move it to the test_programs dir. 

we could change the name of test_programs because in this case it's acting more as a staging directory but the user doesn't actually see the name of the dir so i didn't bother changing it thus far.

Reply:
Okay I didnt know thats how you set it up, we should change the name to just `/etc/hobby-hub/programs` though

Reply:
ok. for now lets NOT make this change until both of our branches are merged in. Afterwards we can make a quick PR changing it. I'll make a git issue for this.